backend/.gitignore

```diff
@@ -1,2 +1,3 @@
 dist/*
 node_modules/*
+.env
```

Comment:
Use the .env file to store environment variables for accessing the database connection. We will each need our own copy, because we will have different passwords, so don't save to git